packages/flutter/lib/src/services/haptic_feedback.dart

```diff
@@ -21,7 +21,7 @@ class HapticFeedback {
   /// On Android, this uses the platform haptic feedback API to simulate a
   /// response to a long press (`HapticFeedbackConstants.LONG_PRESS`).
   static Future<void> vibrate() async {
-    await SystemChannels.platform.invokeMethod('HapticFeedback.vibrate');
+    return SystemChannels.platform.invokeMethod('HapticFeedback.vibrate');
```

Comment:
return seems conceptually wrong here. We're in an `async` function, so we should return a real value, not a Future. (I know that technically an async body has a return value of `FutureOr<T>`, not `T`, but it seems conceptually wrong to do that IMHO.)

Reply:
The nice thing about this one is that type inference gives you `void` for free, but I don't have a strong opinion myself. happy to change it back.

Reply:
Done